src/stabilization_report_template.md

```diff
@@ -0,0 +1,54 @@
+# Stabilization report template
+
+> **What is this?** This is a template to use for [stabilization reports](./stabilization_guide.md). It consists of a series of questions that aim to provide the information most commonly needed and to help reviewers be more likely to identify potential problems up front. Not all parts of the template will apply to all stabilizations. Feel free to put N/A if a question doesn't seem to apply to your case.
+
+## General design
+
+### What is the RFC for this feature and what changes have occurred to the user-facing design since the RFC was finalized?
+
+### What behavior are we committing to that has been controversial? Summarize the major arguments pro/con.
+
+### Are there extensions to this feature that remain unstable? How do we know that we are not accidentally committing to those?
+
+## Has a call-for-testing period been conducted? If so, what feedback was received?
```

Comment:
Maybe it would be useful to mention any OSS nightly users there. *usually* when a feature works, feedback is quite scant but "searched grep.app for `#![feature(...)]` and had N results" gives a good gauge I think.

Reply:
Added this remark